gnucash/gnome-utils/gnc-gtk-utils.c

```diff
@@ -297,3 +297,51 @@ gnc_style_context_get_border_color (GtkStyleContext *context,
     *color = *c;
     gdk_rgba_free (c);
 }
+
+struct about_data
```

Comment:
maybe this shouldn't be named `about_data` because it applies to any container. This file's functions iterates all children even if the widget has been found. Shouldn't it return early if widget is found?

Reply:
I think I missed changing it when I moved the function to gnc-gtk-utils, so maybe change to `dialog_data`. As for finishing early if found, I did not think to do so as there weren't very many widgets and the rarity of use but if you think it worth doing, change it.